src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
```

Comment:
because this field not be changed
```suggestion
  shape = 'triangle'
```